contrib/nextcloud-sync/template.py

```diff
@@ -0,0 +1,34 @@
+pkgname = "nextcloud-sync"
+pkgver = "3.14.0"
+_pkgver = f"{pkgver}-rc1"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = [
+    "cmake",
+    "inkscape",
+    "ninja",
+    "openssl-devel",
+    "pkgconf",
+]
+makedepends = [
+    "karchive-devel",
+    "kguiaddons-devel",
+    "qt6-qt5compat-devel",
+    "qt6-qtsvg-devel",
+    "qt6-qttools-devel",
+    "qt6-qtwebengine-devel",
+    "qt6-qtwebsockets-devel",
+    "qtkeychain-devel",
+    "sqlite-devel",
+    "zlib-ng-compat-devel",
+]
+pkgdesc = "Desktop sync client for Nextcloud"
+maintainer = "jabuxas <jabuxas@proton.me>"
+license = "GPL-2.0-only"
+url = "https://github.com/nextcloud/desktop"
+source = f"{url}/archive/refs/tags/v{_pkgver}.tar.gz"
+sha256 = "21089b8408f3a7a40ad5db3bf9096b437e2e96460e32e541063454ba52115351"
+
+
+def post_install(self):
+    self.install_license("COPYING")
```

Comment:
gpl licences don't have to be installed

Reply:
well it's because it has a license and a copying but sure

Reply:
oh yea true, i thought copying was something else, didnt see that it was gpl2 inside of it